pkg/rancher-desktop/utils/_demo_metadata.js

```diff
@@ -1661,4 +1661,43 @@ export default {
       },
     ],
   },
+  'ghcr.io/rancher-sandbox/rancher-desktop-rdx-open-webui': {
```

Comment:
Since you added the extension at the top in `_demo_marketplace_items.js`, shouldn't it be at the top here too? It will be easier to maintain if extensions are included in the same order in both files.

Reply:
Moved the config to the top of the file.